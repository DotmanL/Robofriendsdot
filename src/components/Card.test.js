
import React from 'react'
import { shallow } from 'enzyme';
import Card from './Card';
import toJson from "enzyme-to-json";

//enzyme testing
/*it("expect to render card component", () =>{
expect(shallow(<Card />).length).toEqual(1)
})
*/


//snapshot testing
it("expect to return Card component", () => {
    const wrapper = shallow(<Card />);
    expect(
      toJson(wrapper)
    ).toMatchSnapshot();
  });