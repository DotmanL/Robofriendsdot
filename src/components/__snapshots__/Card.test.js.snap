// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to return Card component 1`] = `
<div
  className=" tc bg-white dib br3 pa3 ma2 grow bw2 shadow-5"
>
  <img
    alt="robots"
    src="https://robohash.org/undefined?200x200"
  />
  <div>
    <h2 />
    <p />
  </div>
</div>
`;
